internal/admission/server.go

```diff
@@ -7,7 +7,7 @@ import (
 	"io/ioutil"
 	"net/http"
 
-	configuration "github.com/kong/kubernetes-ingress-controller/pkg/apis/configuration/v1"
+	configuration "github.com/kong/kubernetes-ingress-controller/railgun/apis/configuration/v1"
```

Comment:
note: this is where you'll start to see that this PR is converging the two public APIs rather than having separate "duplicate" APIs. These APIs are entirely compatible, and having the separation creates a variety of type related problems which are thrown out the door by simply deduplicating these.